### ISSUE TITLE
Embedding layer distillation not implemented?

### ISSUE THREAD

**Issue by aarmstrong78** [huawei-noah/Pretrained-Language-Model#43]
Discussed in the paper and included in results, but I can't see this referenced in the Readme or anywhere in the code.  Was it implemented in a later (unreleased) version?

**Comment by chuanhuayang** [comment]
embedding layer distillation use the same loss function  MSE as hidden state layer.
so, the embedding layer distillation loss compute is same with hidden state. the code is at 958~960 line of task_distill.py（new_student_reps[0], new_teacher_reps[0]  are the embedding layer output）。

**Comment by silencio94** [comment]
@chuanhuayang 
I think he's talking about "general distillation". general distillation code dose not include embedding-layer distillation. but paper include it.

**Comment by liebkne** [comment]
There's indeed no such thing called 'embedding-layer distillation'.

**Comment by ailinbest** [comment]
sequence_output of the TinyBertForSequenceClassification includes the Embedding layer's weights
